tests/testdata/unit/domain_resolver/rules/generic/domain_resolver.yml

```diff
@@ -26,10 +26,3 @@ filter: client_2
 domain_resolver:
   source_url_or_domain: client_2
 description: ""
----
```

Comment:
Why does the `repr` method alone make some tests fail? And how come just removing this is the right way to solve the problem?

Reply:
because we had a bug so the domain_resolver in this case loads same rules twice?

Reply:
okay